pom.xml

```diff
@@ -4990,6 +4990,9 @@
                 <param>
                   com.puppycrawl.tools.checkstyle.checks.FinalParametersCheckTest
                 </param>
+                <param>
+                  com.puppycrawl.tools.checkstyle.checks.coding.MissingSwitchDefaultCheckTest
+                </param>
               </targetTests>
```

Comment:
like this ? @mahfouz72

Reply:
yes pitest should be green now.
@romani please approve other workflows

Reply:
Done